manifests/plugin/python.pp

```diff
@@ -20,26 +22,39 @@
   }
   # End deprecation
 
-  file {
-    "${name}.load":
-      ensure  => $ensure,
-      path    => "${conf_dir}/${order}-${name}.conf",
-      owner   => 'root',
-      group   => $collectd::params::root_group,
-      mode    => '0640',
-      content => template('collectd/python.conf.erb'),
-      notify  => Service['collectd'],
-  }
-
-  file {
-    "${name}.script":
-      ensure  => $ensure,
-      path    => "${modulepath}/${module}.py",
-      owner   => 'root',
-      group   => $collectd::params::root_group,
-      mode    => '0640',
-      source  => $script_source,
-      require => File["${name}.load"],
-      notify  => Service['collectd'],
+  $ensure_modulepath = $ensure ? {
+      'absent' => $ensure,
+      default  => 'directory',
+  }
+
+  file { $modulepath :
+      ensure  => $ensure_modulepath,
+      mode    => '0755',
```

Comment:
Probably not, but it should not contain sensitive stuff, there are config files for that.